app/controllers/labs_controller.rb

```diff
@@ -173,7 +174,7 @@ def lab_params
       :programs,
       :improve_approval_application,
       capabilities: [],
-      facilities_attributes: [:id, :thing_id, '_destroy'],
```

Comment:
Do you think it is OK to simply remove this?
Did you check if other parts of the app are using this?

Reply:
I've checked the exact line facilities_attributes: [:id, :thing_id, '_destroy'] and there is only the labs form using it. However, things controller also has this param and machines views are using it but I saw no strange behaviors :-?